macheteLogger/src/main/java/com/virtuslab/logger/MacheteLogger.java

```diff
@@ -5,30 +5,73 @@
 import kr.pe.kwonnam.slf4jlambda.LambdaLogger;
 
 public class MacheteLogger implements IMacheteLogger {
-  private final LambdaLogger LOG;
+  private final LambdaLogger logger;
   private final String className;
 
   MacheteLogger(LambdaLogger logger) {
-    LOG = logger;
-    // We are sure that at the moment when this constructor is called we have at least 4 elements in stacktrace array
+    this.logger = logger;
+    // We are sure that at the moment when this constructor is called we have at least 4 elements in stacktrace array:
+    // #0: java.lang.Thread#getStackTrace
+    // #1: com.virtuslab.logger.MacheteLogger#<init>
+    // #2: com.virtuslab.logger.MacheteLoggerFactory#getLogger
+    // #3: method from MacheteLoggerFactory#getLogger was called
     @SuppressWarnings("upperbound")
     StackTraceElement element = Thread.currentThread().getStackTrace()[3];
     String[] classPath = element.getClassName().split("\\.");
     className = classPath[classPath.length - 1];
   }
 
-  private String getMethodReferenceName() {
-    // We are sure that at the moment when this method is called we have at least 7 elements in stacktrace array
+  private String getLogMessagePrefix() {
+    // We are sure that at the moment when this method is called we have at least 7 elements in stacktrace array:
+    // #0: java.lang.Thread#getStackTrace
+    // #1: com.virtuslab.logger.MacheteLogger#getMethodReferenceName
+    // #2: com.virtuslab.logger.MacheteLogger#lambda$debug$0
+    // #3: kr.pe.kwonnam.slf4jlambda.defaultlogger.LambdaLoggerLocationAwareImpl#doLog
```

Comment:
Okay... the problem here is that if/when `slf4jlambda` changes anything than we can have a `surprise` (we're effectively relying on its implementation and not interface now)... but I think that's quite unlikely anyway that this library will get updated soon/ever :) Actually, the usefulness of `slf4jlambda` now is so minuscule that I think you could simply reimplement it here yourself :laughing:

Reply:
Ok, I can try  :smiley:

Reply:
Actually, I leave present logger as is - it will be refactored in future PR